source/WebConfig/webconfig_internal.c

```diff
@@ -437,11 +431,17 @@ int handleHttpResponse(long response_code, char *webConfigData, int retry_count,
 	if((response_code !=403) && (first_digit == 4)) //4xx
 	{
 		WebConfigLog("Action not supported. response_code:%d\n", response_code);
+		addWebConfigNotifyMsg(configURL, response_code, NULL, 0, RequestTimeStamp , "v1.0-NONE");
```

Comment:
How come all cases its "NONE", get of old version should be used.

Reply:
Incorporated.